### PR TITLE
Modal Pop Window

### PR DESCRIPTION
**Pull request by Kwentsir**
In this project, the following were implemented:

- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.

- When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

**Comment by ShoiraTa** [comment]
Hi  👋🏻 ,
While you made a great effort in this project, unfortunately, I cannot proceed to review your code. 😢 

### Invalid Code Review Request

-This is a pair programming activity, but there is only 1 contributor. Please make sure that both participants make contributions. 

 **Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready pray** 

Cheers and Happy coding! 👏🏻👏🏻 

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear._

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

***

_Invalid Code Review Request Does not count into the code reviews limit._

File: script.js [hunk 1]
```diff
@@ -1,127 +1,123 @@
-const navMenu = document.querySelector(".mobile-list");
-const navItems = document.querySelectorAll(".nav-items");
-const closeNav = document.querySelector(".close-nav");
-const humburger = document.querySelector(".union");
+const navMenu = document.querySelector('.mobile-list');
```
Comment (MeqdamAlqudah) [inline script.js:1]: - [ ] Please make sure that you have 4 checks for linters for HTML And CSS and JavaScript 
image

File: script.js [hunk 2]
```diff
@@ -146,24 +142,24 @@ function myWorkSection() {
...
 function popUp() {
-  const mq = window.matchMedia("(min-width: 67.5rem)");
+  const mq = window.matchMedia('(min-width: 67.5rem)');
   if (mq.matches) {
     // window width is at  1080 px or more
-    const desktop = document.querySelector(".desktop-popcontainer");
+    const desktop = document.querySelector('.desktop-popcontainer');
     desktop.innerHTML = `
     <div class="desktop-popclass" id="desktop-popid">
     <div class="desktop-popimages">
```
Comment (MeqdamAlqudah) [inline script.js:165]: - [ ] Please add a margin to the top and to the bottom of the popup window so it matches Figma design 

| Your design | Figma |
| ------------- | ------- |
| image
image | image |

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @Kwentsir , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by aimemalaika — APPROVED** [review]
Hi @Kwentsir , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**